packages/seeder/src/manufacturer/manufacturer.model.ts

```diff
@@ -0,0 +1,62 @@
+import { model, Schema } from 'mongoose';
+import { Manufacturer } from 'models';
+
+interface ManufacturerWithID extends Manufacturer {
+    id: {
+        type: String,
+        unique: true
+    }
```

Comment:
id: string; would've been enough:--)

Reply:
Ohh, yes. I see!

Reply:
not updated yet. and also please update the seeder code with new convention. just add a default export in model file. for example check new-tehcstack-monorepo-backup branch.